src/components/FishImg.vue

```diff
@@ -1,14 +1,22 @@
 <template>
-  <span v-if="invert === 'true'" class="invert">
-    <b-img :src="src" :name="name" :title="title" :height="height" :width="width" fluid :alt="name" />
-  </span>
-  <span v-else>
-    <b-img :src="src" :name="name" :title="title" :height="height" :width="width" fluid :alt="name" />
-  </span>
+  <HubImg
+    v-if="invert === 'true'"
+    :src="src"
+    :name="name"
+    :title="title"
+    :height="height"
+    :width="width"
+    class="invert"
+    fluid
+    :alt="name"
+  />
+  <HubImg v-else :src="src" :name="name" :title="title" :height="height" :width="width" fluid :alt="name" />
```

Comment:
let's use the [class & style bindings](https://vuejs.org/v2/guide/class-and-style.html) so we don't have to use the if-else here....

```html
  <HubImg
    :src="src"
    :name="name"
    :title="title"
    :height="height"
    :width="width"
    :class="{invert: invert === 'true'}"
    fluid
    :alt="name"
  />
```